clang/test/CodeGenCXX/cxx23-p2280r4.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clang_cc1 -triple %itanium_abi_triple -std=c++23 %s -emit-llvm -o - | FileCheck %s
```

Comment:
This should be tested in old modes to prevent potential regressions.

P2280R4 is a defect report (ideally, against C++11/14/17/20), although Clang only applies it in C++23 and later currently, see also #127166. I think Clang's choice is currently reasonable, but it should changed one day.